cfgov/v1/templatetags/share.py

```diff
@@ -21,9 +22,10 @@ def is_shared(page):
 @register.assignment_tag(takes_context=True)
 def staging_url(context, page):
```

Comment:
looks like this is determining which url to use either live/shared urls. maybe lets rename the method to something clearer?
e.g `live_shared_url`, `get_page_url`